src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/hcaptcha/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    cq:icon="shield"
+    jcr:primaryType="cq:Component"
+    jcr:title="Adaptive Form hCaptcha® (v1)"
+    sling:resourceSuperType="core/fd/components/form/hcaptcha/v1/hcaptcha"
+    componentGroup="Core Components Examples - Adaptive Form"/>
```

Comment:
Component group should be variable. Please refer https://github.com/adobe/aem-project-archetype/blob/develop/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/checkbox/.content.xml#L7